src/js/models/v2/awardsV2/BaseAwardAmounts.js

```diff
@@ -29,11 +29,16 @@ const BaseAwardAmounts = {
         return MoneyFormatter.formatMoneyWithPrecision(this._obligation, 2);
     },
     get obligationFormatted() {
-        // TODO - Lizzie: handle negative values
-        if (this._obligation >= MoneyFormatter.unitValues.MILLION) {
+        if (Math.abs(this._obligation) >= MoneyFormatter.unitValues.MILLION) {
             const units = MoneyFormatter.calculateUnitForSingleValue(this._obligation);
+            if (this._obligation < 0) {
+                return `(${MoneyFormatter.formatMoneyWithPrecision(Math.abs(this._obligation) / units.unit, 1)} ${units.unitLabel})`;
```

Comment:
Could you walk me through what's happening here for my future reference? Specifically how this function handles the first parameter `absolute value / Million`?

Reply:
Also I see we're not returning a negative value here. I guess there is a check for if the `.obligationFormatted()` should be negative elsewhere.

Would like to hear your thought process around this choice as to why the return value of this function is absolute!

Reply:
Good question - the [mockup for de-obligated](https://bahdigital.invisionapp.com/share/69IA8EPGPCM#/296009661_combinedAwardsSection--deobligation) scenario is displaying negative values that way. And the width of that diagonal stripe section will represent the absolute value of the negative obligations. The actual/negative value will also still be available via the `_obligation` property.

Reply:
Sounds good.